src/lock.ts

```diff
@@ -1,16 +1,14 @@
 import { N9Error, waitFor } from '@neo9/n9-node-utils';
+import * as crypto from 'crypto';
 import * as _ from 'lodash';
 import * as mongodb from 'mongodb';
-import * as mongoDbLock from 'mongodb-lock';
+import { LangUtils } from '.';
```

Comment:
```suggestion
import { LangUtils } from './lang-utils';
```
avoid loops in imports